.github/workflows/generate-binaries.yaml

```diff
@@ -48,6 +46,33 @@ jobs:
 
       - uses: actions/upload-artifact@v3
         with:
-          name: MacOS ISPC kernels
+          name: ISPC kernels
           path: |
             src/ispc/libdownsample_ispcx86_64-apple-darwin.a
+
+  build-windows:
+    runs-on: windows-latest
+
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          submodules: true
+
+      - name: Clean old artifacts before upload globbing
+        run: Remove-Item src/ispc/*.lib
+
+      - name: Prepare Environment
+        run: |
+          curl -OL https://github.com/ispc/ispc/releases/download/v1.17.0/ispc-v1.17.0-windows.zip
```

Comment:
I suppose we have to keep updating it with the right version? Might be worth a comment?

Reply:
Yeah, this was going to be part of a followup push :)